src/App.js

```diff
@@ -1,8 +1,72 @@
+import { React, useState, useReducer, useEffect } from "react";
+import { FaBookReader } from "react-icons/fa";
+
+import InputContainer from "./containers/InputContainer";
+import ListItemContainer from "./containers/ListItemContainer";
+
+//로컬 스토리지의 정보구성 {list: {item1, item2, ...}}
+
+const initList = localStorage.getItem("list")
+  ? JSON.parse(localStorage.getItem("list"))
+  : [];
```

Comment:
[short-circuit evaluation](https://velog.io/@nudge411/%EB%8B%A8%EC%B6%95-%ED%8F%89%EA%B0%80-short-circuit-evaluation-%EB%85%BC%EB%A6%AC-%EA%B3%84%EC%82%B0%EB%B2%95)을 활용해서 더 간결하게 써 볼 수도 있을 것 같아요~ 

```suggestion
const initList = JSON.parse(localStorage.getItem('list')) || [];
```

Reply:
short circuit evaluation은 처음 보는 문법이에요! 알려주셔서 감사합니다. 참고하도록 하겠습니다